### ISSUE TITLE
the configuration of t-SNE visualization

### ISSUE THREAD

**Issue by sndnyang** [P2333/Max-Mahalanobis-Training#3]
In your paper, you have very dense clusters in your t-SNE visualization plot.

![image](https://user-images.githubusercontent.com/2310591/91078730-0b801f80-e611-11ea-96a1-782976a2cee1.png)

I can generate very dense logits in 10-dimension to the true center.  
the features/outputs after the LDA  like   [100+, 100+, 100+,  0.1-,  100+, 100+, ...]. It's very close to the center in 10d.
But when I apply t-SNE (with sklearn library),  it generates something like sticks/bands.

![image](https://user-images.githubusercontent.com/2310591/91079157-af69cb00-e611-11ea-82e4-79af469708d0.png)

So any suggestion to generate the t-SNE visualization better as yours?

**Comment by sndnyang** [comment]
OK, I made mistakes. Solve it. I should use the features,  not logits~~~

**Comment by sndnyang** [comment]
OK, I made mistakes. Solve it. I should use the features,  not logits~~~
